invenio_app_ils/ui/backoffice/src/index.scss

```diff
@@ -0,0 +1,14 @@
+body {
```

Comment:
https://emotion.sh/ 😎

Reply:
this is coming from the bootstrapping of the project. Didn't use any styling library yet.